src/librustc_parse/parser/item.rs

```diff
@@ -568,7 +568,7 @@ impl<'a> Parser<'a> {
             && self.look_ahead(1, |t| t.is_non_raw_ident_where(|i| i.name != kw::As))
         {
             self.bump(); // `default`
-            Defaultness::Default(self.prev_span)
+            Defaultness::Default(self.normalized_prev_token.span)
```

Comment:
This is because of the `is_non_raw_ident_where`?

Reply:
No, because `default` is an identifier.

It's more important for `async` which performs an edition check on the span, but I used normalization for all `enum Foo { Yes(Span), No }` keywords for consistency.